src/__storybook_helpers__/components/app-layout/app-layout.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
```

Comment:
rather than this directory being in `/src`, can it be moved into the `.storybook` directory - that way all the storybook config is in one place, and everything in src gets compiled and served in lib which could be confusing.

Reply:
sure, fixed, thanks